### ISSUE TITLE
L_1 norm is calculated incorrectly in the MIFGSM method

### ISSUE THREAD

**Issue by huitailangyz** [Harry24k/adversarial-attacks-pytorch#12]
The  L1 norm in MIFGSM now is calculated per batch, but actually is should be calculated per image. 
https://github.com/Harry24k/adversarial-attacks-pytorch/blob/3613a1a90946f58d48450f7f11702ad979162a35/torchattacks/attacks/mifgsm.py#L56 
The implementation now leads to different results when using different batch size.
For my evaluation (used my own trained model in CIFAR10), the difference of the accuracy between the batch size 1 vs 100 is about 1%.

**Comment by Harry24k** [comment]
Hello!
Thanks for your correction.
Please refer to [v2.11.0](https://github.com/Harry24k/adversarial-attacks-pytorch/commit/82d5b827832855d7cb201785c6d958810e8c11f8).
